pkg/dashboard/ui/src/app/app.run.js

```diff
@@ -37,5 +38,50 @@
         // @endif
 
         /*eslint angular/on-watch: 0*/
+
+        i18next
+            .use($window.i18nextChainedBackend)
+            .use($window.i18nextBrowserLanguageDetector);
+
+        i18next.init({
+            debug: false,
+            fallbackLng: 'en',
+            preload: ['en'],
+            initImmediate: false,
+            nonExplicitWhitelist: true,
+            partialBundledLanguages: true,
+            defaultNs: 'common',
+            ns: [
+                'login',
+                'common',
+                'header',
+                'home',
+                'services',
+                'containers',
+                'logs',
+                'clusters',
+                'networks',
+                'identity',
+                'tasks',
+                'tenants',
+                'events',
+                'storagePools',
+                'functions'
```

Comment:
No, it doesn't make sense that open-source Nuclio UI project that should be operational without Iguazio would have all these namespaces.